build_tools/update_environments_and_lock_files.py

```diff
@@ -198,12 +198,19 @@ def remove_from(alist, to_remove):
         "folder": "build_tools/azure",
         "platform": "linux-64",
         "channel": "conda-forge",
-        "conda_dependencies": ["pypy"]
+        "conda_dependencies": ["pypy", "python"]
         + remove_from(
             common_dependencies_without_coverage, ["python", "pandas", "pillow"]
         )
         + ["ccache"],
-        "package_constraints": {"blas": "[build=openblas]"},
+        "package_constraints": {
+            "blas": "[build=openblas]",
+            # XXX: this can be removed when the vendored cloudpickle inside
+            # joblib includes the cloudpickle fix from
+            # https://github.com/cloudpipe/cloudpickle/pull/469
+            # 2022-09-06: the fix is not in any cloudpickle release yet
```

Comment:
Coming from https://github.com/cloudpipe/cloudpickle/pull/461#issuecomment-1180900147, I do not know if the fix is simple.

Reply:
I tweak the comment to indicate that the issue is more broad. Also I debugged the "worst"  cloudpickle PyPy > 3.7 errors so hopefully that helps having a full fix one day in cloudpickle and consequently removing this pin in the not too distant future :crossed_fingers: